backend/src/main/java/com/huellapositiva/infrastructure/SecurityConfig.java

```diff
@@ -23,12 +25,25 @@ PasswordEncoder passwordEncoder() {
         return new BCryptPasswordEncoder();
     }
 
+    @Bean
+    public WebMvcConfigurer corsConfigure(){
+        return new WebMvcConfigurer() {
+            @Override
+            public void addCorsMappings(CorsRegistry registry) {
+                registry.addMapping("/api/v1/**")
+                        .allowedOrigins("http://localhost:3000")
+                        .allowedMethods("GET", "POST", "PUT", "DELETE")
```

Comment:
Is it worth allowing HEAD and OPTIONS methods? Check if your frontend js framework requires it, I think Angular for instance needed it

Reply:
It's not necessary for react, u are right. Remove the allowedMethods